### PR TITLE
RobinRound.cpp

### PR DESCRIPTION
**Pull request by Mridul07Sharma**

### CODE REVIEW

**Comment by heyimShivam** [comment]
Please indent  the code.

**Comment by Mridul07Sharma** [comment]
@heyimShivam it is already indented.

File: RobinRound.cpp [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+
+void findWaitingTime(int processes[], int n, 
+
+            int bt[], int wt[], int quantum) 
+{ 
+
+    // Make a copy of burst times bt[] to store remaining 
+
+    // burst times. 
+
```
Comment (heyimShivam) [inline RobinRound.cpp:17]: no need for new line after comment

File: RobinRound.cpp [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+    int rem_bt[n]; 
+
+    for (int i = 0 ; i < n ; i++) 
+
+        rem_bt[i] = bt[i]; 
+ 
+
```
Comment (heyimShivam) [inline RobinRound.cpp:24]: no need

File: RobinRound.cpp [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+    int t = 0; // Current time 
+ 
+
+    // Keep traversing processes in round robin manner 
+
+    // until all of them are not done. 
+
```
Comment (heyimShivam) [inline RobinRound.cpp:31]: no need of blank line

File: RobinRound.cpp [hunk 4]
```diff
@@ -0,0 +1,215 @@
...
+    while (1) 
+
```
Comment (heyimShivam) [inline RobinRound.cpp:33]: no need of blank line

File: RobinRound.cpp [hunk 5]
```diff
@@ -0,0 +1,215 @@
...
+    { 
+
+        bool done = true; 
+ 
+
+        // Traverse all processes one by one repeatedly 
+
+        for (int i = 0 ; i < n; i++)
```
Comment (heyimShivam) [inline RobinRound.cpp:41]: no need of blank line

File: RobinRound.cpp [hunk 6]
```diff
@@ -0,0 +1,215 @@
...
+
+            // If burst time of a process is greater than 0 
+
+            // then only need to process further 
+
+            if (rem_bt[i] > 0) 
+
+            { 
+
+                done = false; // There is a pending process
```
Comment (heyimShivam) [inline RobinRound.cpp:53]: ``
```suggestion
            if (rem_bt[i] > 0) { 
                done = false; // There is a pending process 
```

File: RobinRound.cpp [hunk 7]
```diff
@@ -0,0 +1,215 @@
...
+                    // by quantum 
+
+                    rem_bt[i] -= quantum; 
+
+                } 
+ 
+
+                // If burst time is smaller than or equal to 
+
+                // quantum. Last cycle for this process
```
Comment (heyimShivam) [inline RobinRound.cpp:78]:
```suggestion
                if (rem_bt[i] > quantum) { 
                    // Increase the value of t i.e. shows 
                    // how much time a process has been processed 
                    t += quantum; 
                    // Decrease the burst_time of current process 
                    // by quantum 
                    rem_bt[i] -= quantum; 
                } 
                // If burst time is smaller than or equal to 
                // quantum. Last cycle for this process 
```

**Review by heyimShivam — CHANGES_REQUESTED** [review]
Please correct indentation in whole file

**Review by Mridul07Sharma — COMMENTED** [review]
Changes done

**Review by heyimShivam — APPROVED** [review]
LGTM, Thanks for the changes

**Pull request merged**